networking/ipsec/ipsec_basic/ipsec_basic_netns/runtest.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+# vim: dict=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#	runtest.sh of /kernel/networking/ipsec/ipsec_basic/ipsec_basic_netns
+#	Description: ipsec/ipsec_basic/ipsec_basic_netns
+#	Author: Xiumei Mu <xmu@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#	Copyright (c) 2016 Red Hat, Inc. All rights reserved.
+#
+#	This copyrighted material is made available to anyone wishing
+#	to use, modify, copy, or redistribute it subject to the terms
+#	and conditions of the GNU General Public License version 2.
+#
+#	This program is distributed in the hope that it will be
+#	useful, but WITHOUT ANY WARRANTY; without even the implied
+#	warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#	PURPOSE. See the GNU General Public License for more details.
+#
+#	You should have received a copy of the GNU General Public
+#	License along with this program; if not, write to the Free
+#	Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+#	Boston, MA 02110-1301, USA.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Global parameters
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+. ../../../common/include.sh
```

Comment:
This might be a stupid question but does it work? We had some discussions with Cestmir because he used something similar (include in the runtest but the files were not mentioned in the makefile) and the downloaded test then didn't contain the includes because they were in a different directory. I don't remember if this is the same setup as he has but I thought I'd ask :)

Reply:
@veruu it does work :-) since we are using restraint, we need to include a metadata file and include repoRequires, otherwise libraries will not be extracted during the fetch from github: https://restraint.readthedocs.io/en/latest/tasks.html#softdependencies
see https://github.com/CKI-project/tests-beaker/pull/189/files#diff-ebc14df65bbe14531bab7b4dc049900bR13

Reply:
Oh the reporequires is in metadata! Ok, thanks for explanation!